src/components/App.tsx

```diff
@@ -2240,27 +2238,51 @@ class App extends React.Component<AppProps, AppState> {
     target:
       | ExcalidrawElement
       | readonly ExcalidrawElement[] = this.scene.getNonDeletedElements(),
-    opts?: { fitToContent?: boolean; animate?: boolean; duration?: number },
+    opts?:
+      | {
+          fitToContent?: boolean;
```

Comment:
we should rename to something like `fitContent`. `fitToContent` sounds it should work like `fitToViewport` currently does. TBD.